src/cls/rbd/cls_rbd_client.cc

```diff
@@ -149,6 +149,18 @@ namespace librbd {
                                          parent);
     }
 
+    void create_image(librados::ObjectWriteOperation *op, uint64_t size, uint8_t order,
```

Comment:
The sync versions do use the async helpers which got introduced in the prior commit. This commit changes the sync version to use async helpers.